### ISSUE TITLE
allow each responder to create a dbus server

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#3143]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/3148
- Created at 2016-08-22 12:44:16 by [pbrezina](https://pagure.io/user/pbrezina)
- Closed at 2020-03-24 14:04:23 as wontfix
- Assigned to [pbrezina](https://pagure.io/user/pbrezina)
---

Each responder should be able to create a D-Bus server similar as data provider so they can contact each others. Responders should be reachable especially from ifp.

### Comments

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2016-08-22 12:56:51](https://pagure.io/SSSD/sssd/issue/3148#232620)**

Fields changed

owner: somebody => pbrezina
status: new => assigned

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2016-08-29 11:45:04](https://pagure.io/SSSD/sssd/issue/3148#232621)**

Deferring until we need this atually.

milestone: NEEDS_TRIAGE => SSSD Deferred

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2016-08-30 10:29:10](https://pagure.io/SSSD/sssd/issue/3148#232622)**

Needed for #3147

milestone: SSSD Deferred => SSSD 1.15 Beta

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2016-08-31 16:46:35](https://pagure.io/SSSD/sssd/issue/3148#232623)**

Fields changed

rhbz:  => 0

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2017-02-24 14:46:04](https://pagure.io/SSSD/sssd/issue/3148#232624)**

**Metadata Update from @pbrezina**:
- Issue assigned to pbrezina
- Issue marked as blocked by: #3147
- Issue marked as blocked by: #3149
- Issue set to the milestone: SSSD 1.15.3

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2017-03-31 18:08:48](https://pagure.io/SSSD/sssd/issue/3148#434529)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from 0)
- Custom field mark reset (from 0)
- Custom field patch reset (from 0)
- Custom field review reset (from 0)
- Custom field sensitive reset (from 0)
- Custom field testsupdated reset (from 0)
- Issue **un**marked as depending on: #3149

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2017-03-31 18:11:55](https://pagure.io/SSSD/sssd/issue/3148#434538)**

**Metadata Update from @jhrozek**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue set to the milestone: SSSD Future releases (no date set yet) (was: SSSD 1.15.3)

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2020-03-13 11:32:56](https://pagure.io/SSSD/sssd/issue/3148#633037)**

**Metadata Update from @thalman**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field review reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue tagged with: Canditate to close

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2020-03-24 14:04:22](https://pagure.io/SSSD/sssd/issue/3148#636455)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2020-03-24 14:04:24](https://pagure.io/SSSD/sssd/issue/3148#636456)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
